Certificate.py

```diff
@@ -0,0 +1,80 @@
+import json
+import ntpath
+from typing import Optional, Dict, List
+
+from Enum.JsonStructureKeys import JsonStructureKey
+from JsonItems.IJsonItem import IJsonItem
+
+
+# Class representation of certificate
+# it contains content, filename, content per page
+# content per page is guessed based on footers
+class Certificate:
+    def __init__(self, filename: str, content: str):
+        self._filename = ntpath.basename(filename)
+        # backup file name
+        if not self._filename:
+            self._filename = filename
+        self._content = content
+        self._pages = self._get_per_pages()
+        self._json = {}
+
+    def _get_per_pages(self) -> Dict[int, str]:
+        pages = {}
+        page_number = 1
+        splited_pages = self._content.split('\f')
+        for page_content in splited_pages:
+            if page_number < len(splited_pages):
+                pages[page_number] = page_content
+                page_number += 1
+        return pages
```

Comment:
Using dictonary for storing pages seems unnecesary - it introduces potentional slowdown.
Using a list instead would be better in this case, since you are just indexing using page numbers.